recharts-bar-chart/src/App.js

```diff
@@ -0,0 +1,145 @@
+import {
+  client,
+  useConfig,
+  useElementData,
+  useElementColumns,
+} from "@sigmacomputing/plugin";
+import "./App.css";
+import React from "react";
+import {
+  ResponsiveContainer,
+  BarChart,
+  Bar,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+} from "recharts";
+
+client.config.configureEditorPanel([
+  { name: "source", type: "element" },
+  {
+    name: "dimension",
+    type: "column",
+    source: "source",
+    allowMultiple: false,
+  },
+  {
+    name: "measures",
+    type: "column",
+    source: "source",
+    allowMultiple: true,
+  },
+]);
+
+function RenderBarChart(props) {
+  console.log(props);
+  return (
+    <ResponsiveContainer width="200%" height={500}>
```

Comment:
it wasn't wide enough to fit all of the bars for the sample table i was using to test it, but i could decrease it and just mention in a comment somewhere that it might be a tight fit past x number of bars